### PR TITLE
Update README(.md) for 0.7.0

### PR DESCRIPTION
**Pull request by alagoutte**

### CODE REVIEW

**Comment by alagoutte** [comment]
@CedricMoreau can you look

File: README.md [hunk 1]
```diff
@@ -45,12 +45,15 @@ With this module (version 0.6.1) you can manage:
...
 - User Group (Get)
+- User SAML (Get)
 - User RADIUS (Get)
 - [VDOM](#vdom) (Get)
 - [Virtual IP](#virtual-ip) (Add/Get/Remove object type static-nat)
 - [Virtual IP Group](#virtual-ip-group) (Add/Get/Copy/Set/Remove and Add/Remove Member)
 - Virtual WAN Link/SD-WAN (Get)
-- VPN IPsec Phase 1/Phase 2 Interface (Get)
+- [VPN IPsec](#vpn-ipsec) [Phase 1](#vpn-ipsec-interface-phase-1)/[Phase 2](#vpn-ipsec-interface-phase-2) Interface (Add/Get/Set/Remove)
+- VPN SSL (Get Client, Portal, Settings)
```
Comment (CedricMoreau) [inline README.md:55]: No caps on portal and settings ?

File: README.md [hunk 2]
```diff
@@ -106,8 +109,29 @@ The first thing to do is to connect to a FortiGate Firewall with the command `Co
...
-if you get a warning about `Unable to connect` Look [Issue](#issue)
 
+You can select the port using `-port` parameter
+
+```powershell
+# Connect to the FortiGate Firewall using port 4443
+    Connect-FGT 192.0.2.1 -port 4443
+```
+
+if you are using OTP (FortiToken) for admin access, you can use -token_code or -token_prompt for specifity or ask the token/OTP when connect
```
Comment (CedricMoreau) [inline README.md:120]: ...ask the token/OTP when connect**ing**

File: README.md [hunk 3]
```diff
@@ -106,8 +109,29 @@ The first thing to do is to connect to a FortiGate Firewall with the command `Co
...
+
+```powershell
+# Connect to the FortiGate Firewall with ask token
```
Comment (CedricMoreau) [inline README.md:123]: with the token asked

File: README.md [hunk 4]
```diff
@@ -917,6 +941,32 @@ modify its properties `Set-FGTSystemInterface` or delete it `Remove-FGTSystemInt
     mode                                       : static
     [...]
 
+# Create an interface (type LACP)
+    Add-FGTSystemInterface -name PowerFGT_lacp -atype lacp -member port9, port10
```
Comment (CedricMoreau) [inline README.md:945]: typo on type ? type instead of atype ?

File: README.md [hunk 5]
```diff
@@ -917,6 +941,32 @@ modify its properties `Set-FGTSystemInterface` or delete it `Remove-FGTSystemInt
...
+    vrf                                        : 0
+    [...]
+    type                                       : aggregate
+    [...]
+    member                                     : {@{interface-name=port9; q_origin_key=port9}, @{interface-name=port10; q_origin_key=port10}}
+    lacp-mode                                  : active
+    [...]
+
+# Create an interface (type Loopback)
+    Add-FGTSystemInterface -name PowerFGT_lo -loopback -mode static -ip 192.0.2.1 -netmask 255.255.255.0 -allowaccess ping
```
Comment (CedricMoreau) [inline README.md:959]: loopback must be /32 ?

**Pull request merged**